features/scv/api/src/main/java/org/opennms/features/scv/api/SecureCredentialsVault.java

```diff
@@ -21,10 +21,79 @@
  */
 package org.opennms.features.scv.api;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Optional;
+import java.util.Properties;
 import java.util.Set;
+import java.util.stream.Stream;
 
 public interface SecureCredentialsVault {
 
+    public enum KeyStoreType {
+        JCEKS,
+        PKCS12;
+
+        public static KeyStoreType fromSystemProperty() {
+            return KeyStoreType.valueOf(System.getProperty(SCV_KEYSTORE_PROPERTY, "JCEKS").toUpperCase());
+        }
+    }
+    public final static String SCV_KEYSTORE_PROPERTY = "org.opennms.features.scv.keystore.type";
+    public final static Properties m_onmsProperties = new Properties();
```

Comment:
Don't need to follow `m_` convention for new changes if there is no such convention in the file already.